tox.ini

```diff
@@ -4,7 +4,7 @@ skipsdist = True
 
 [testenv]
 usedevelop = True
-commands = py.test {posargs:tests/} --assert=plain
```

Comment:
Hum... I only use `tox` to run the full test suite (or to reproduce what the CI system does). If I want to run a specific test, I use pytest directly and specify the filename, such as `py.test some/test/path/test_foo.py`.